subprojects/docs/src/docs/release/notes.md

```diff
@@ -22,6 +22,37 @@ The new metadata format is still under active development, but it can already be
 
 [The JaCoCo plugin](userguide/jacoco_plugin.html) has been upgraded to use [JaCoCo version 0.8.0](http://www.jacoco.org/jacoco/trunk/doc/changes.html) by default.
 
+### Public API for defining command line options for tasks
+
+A custom task implementation exposes properties to make the runtime behavior configurable. Sometimes a user wants to declare the value of an exposed task property on the command line instead of the build script. That's particular helpful if the property value changes more frequently. With this version of Gradle, the task API now supports a mechanism for marking a property to automatically generate a corresponding command line parameter with a specific name at runtime. All you need to do is to annotate a setter method of a property with [Option](dsl/org.gradle.api.tasks.options.Option.html).
```

Comment:
This needs a bit of polish. I think we should reword or remove the first sentence here.

Also "particular" => "particularly"

"That's particular helpful if the property value changes more frequently." is sort of an odd sentence.

Reply:
Can you explain what exactly bothers you about the first sentence? Trying to understand why you think it's needs changing and how. Does it sound too abstract?

Reply:
done